packages/vaadin-ordered-layout/test/visual/lumo/horizontal-layout.test.js

```diff
@@ -0,0 +1,75 @@
+import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
+import { visualDiff } from '@web/test-runner-visual-regression';
+import '../../../theme/lumo/vaadin-horizontal-layout.js';
+
+describe('horizontal-layout', () => {
+  let div, element;
+
+  beforeEach(() => {
+    div = document.createElement('div');
+    div.style.display = 'flex';
+    element = fixtureSync(
+      `
+        <vaadin-horizontal-layout>
+          <div style="background: #e2e2e2; padding: 20px;">Item 1</div>
+          <div style="background: #f3f3f3; padding: 20px;">Item 2</div>
+        </vaadin-horizontal-layout>
+      `,
+      div
+    );
+  });
+
+  it('basic', async () => {
+    await visualDiff(div, 'horizontal-layout:basic');
+  });
+
+  it('theme-margin', async () => {
+    element.setAttribute('theme', 'margin');
+    await visualDiff(div, 'horizontal-layout:theme-margin');
+  });
+
+  it('theme-padding', async () => {
+    element.setAttribute('theme', 'padding');
+    await visualDiff(div, 'horizontal-layout:theme-padding');
+  });
```

Comment:
These two tests look exactly the same. It might be a good idea to set a border of any size and a contrasting color to see a difference between these two.

Reply:
Good catch, done.